datasets/serratus-lovelywater.yaml

```diff
@@ -0,0 +1,34 @@
+Name: Serratus Coronavirus Discovery - Versioned Data Release
```

Comment:
Option for more general use sub-title, "Serratus:  Ultra-deep Search for Novel Viruses -  Versioned Data Release"

Reply:
I like this much better; changing now!